### ISSUE TITLE
Bot question

### ISSUE THREAD

**Issue by b-ma** [orottier/web-audio-api-rs#335]
Could we somehow have a better strategy with the bot?

I'm not sure how to do that because 1. the "on-demand" can (very) rapidly becomes "I don't care / I forgot that thing" but 2. the constant (huge) reporting also creates noise and makes conversations very of hard to follow...

**Comment by b-ma** [comment]
Yup I have used it in some workflow, but had some issues if there is other conditions, e.g. _allow manual trigger_ and _trigger on tag_, probably did something wrong though...

We could also maybe rely on a comment in the PR, cf. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment, e.g. if the message is "[benchmark]" or something run the workflow? Maybe a bit more fluid?

**Comment by orottier** [comment]
To run on comment seems nice as well of course. Feel free to experiment.
FYI, another option is to just not post a comment. The benchmark outputs can also be viewed manually https://github.com/orottier/web-audio-api-rs/actions/runs/5593776474/jobs/10227839885#step:11:65
